js/src/components/Match.vue

```diff
@@ -39,7 +41,9 @@ export default {
   data () {
     return {
       match: new Match(),
-      tournament: new Tournament()
+      tournament: new Tournament(),
+      user: this.$root.user,
+      levels: levels,
```

Comment:
To be honest, it feels a bit dubious, but I couldn't find a better way either :-/

But it works fine and isn't that hard!